src/components/NavBar.js

```diff
@@ -12,7 +12,7 @@ const NavBar = () => {
         <Drawer anchor="left" variant="permanent">
             <StyledDrawer>
                 <ListItem>
-                    <ListItemButton component={RouterLink} to="/create">
+                    <ListItemButton component={RouterLink} to ="/create">
```

Comment:
```suggestion
                    <ListItemButton component={RouterLink} to="/create">
```
This just stay without the whitespace